code/game/machinery/computer/dropship_weapons.dm

```diff
@@ -25,9 +138,24 @@
 	var/datum/simulator/simulation
 	var/datum/cas_fire_mission/configuration
 
+	// groundside maps
+	var/datum/tacmap/tacmap
+	var/minimap_type = MINIMAP_FLAG_USCM
+
+	// Cameras
+	var/datum/camera_manager/cam_manager
+	var/camera_target_id
+	var/camera_width = 11
+	var/camera_height = 11
+
 /obj/structure/machinery/computer/dropship_weapons/Initialize()
 	. = ..()
 	simulation = new()
+	tacmap = new(src, minimap_type)
+
+	// camera setup
+	cam_manager = new("dropship_camera_[REF(src)]_map")
```

Comment:
it's hugely unlikely to matter for the dropship weapons computer but in the future for such situations you may want to consider using something that can't collide under ref reuse, eg. a simple static integer incrementing